summerofcode/README.md

```diff
@@ -6,9 +6,10 @@ The Cloud Native Computing Foundation participates in the [Google Summer of Code
 
 If you need help with anything CNCF + Summer of code related, you can file an issue or contact one of the admins below:
 
-- Chris Aniszczyk ([@caniszczyk](https://github.com/caniszczyk)\): <https://twitter.com/cra>
-- Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi)\): <https://twitter.com/idvoretskyi>
-- Nate Waddington ([@nate-double-u](https://github.com/nate-double-u)\): <https://twitter.com/hackabletype>
+- Chris Aniszczyk ([@caniszczyk](https://github.com/caniszczyk)) - Twitter: [@cra](https://twitter.com/cra), CNCF Slack: [@caniszczyk](https://cloud-native.slack.com/team/U0BB4K6KD)
+- Nate Waddington ([@nate-double-u](https://github.com/nate-double-u)) - Twitter: [@hackabletype](https://twitter.com/hackabletype), CNCF Slack: [@nate-double-u](https://cloud-native.slack.com/team/U01B9CR2A3E)
+- Jay Tiaki ([@jaytiaki](https://github.com/jaytiaki))
+- Ali Ok ([@aliok](https://github.com/aliok)) - CNCF Slack: [@Ali Ok](https://cloud-native.slack.com/team/U030UQH8QNR) 
```

Comment:
Let's:
* remove Chris (Unless you specifically want to stay on this list, @caniszczyk)
* remove individual slack ids
* remove twitter handles

```suggestion
- Nate Waddington ([@nate-double-u](https://github.com/nate-double-u))
- Jay Tiaki ([@jaytiaki](https://github.com/jaytiaki))
- Ali Ok ([@aliok](https://github.com/aliok))
```

We have a communication section later in the README, so I don't think that we need to invite folks to reach out to the admins specifically. It's preferable that they use the open spaces rather than reaching out directly to one of the admis.

Reply:
Done!